### DISCUSSION TITLE
【QA5】Can we host this tool online on any hosting website?

### DISCUSSION THREAD

**Discussion by cooltaby** [smart-test-ti/SoloX#207] [Q&A]
Is it possible to host this tool online and access it from anywhere on internet?

**Comment by rafa0128** [comment]
https://github.com/smart-test-ti/SoloX/issues/198

**Reply by cooltaby** [reply]
so we need an IP which can be accessed online? then we can launch 

also please check your web admin tool, its not working. thanks

**Reply by rafa0128** [reply]
solox-vue-admin is a demo and cannot be directly put into use. You need to continue developing it

**Reply by cooltaby** [reply]
Instead of sharing the IP to connect devices from the host machine to the remote, is it possible to create a new instance of this tool online? This way, we wouldn't need to install it on every machine where it will be used. and everyone can use this tool online for their own devices.

**Comment by rafa0128** [comment]
oh，I misunderstood your question，you can carefully read the README.md, which includes how to deploy it

**Comment by rafa0128** [comment]
> solox-vue-admin is a demo and cannot be directly put into use. You need to continue developing it
You can try using Yarn because NPM always encounters problems

**Reply by cooltaby** [reply]
@rafa0128 I tried with Yarn too, added logs here https://github.com/smart-test-ti/solox-vue-admin/issues/1#issuecomment-1692901594

**Reply by cooltaby** [reply]
Admin tool issue got resolved after downgrading node to 14 [Node.js](https://nodejs.org/en/) ( =14.*)  thanks but still unclear how to connect it with the solox

**Comment by cooltaby** [comment]
> #198
Instead of sharing the IP to connect devices from the host machine to the remote, is it possible to create a new instance of this tool online? This way, we wouldn't need to install it on every machine where it will be used. and everyone can use this tool online for their own devices.
